doc/start/quick-ceph-deploy.rst

```diff
@@ -25,181 +25,137 @@ are in this directory when executing ``ceph-deploy``.
    if you are logged in as a different user, because it will not issue ``sudo``
    commands needed on the remote host.
 
-.. topic:: Disable ``requiretty``
```

Comment:
this is still useful to keep around because it can still happen. Any strong reasons why this needs to be removed?